bundle/manifests/sonataflow-operator.clusterserviceversion.yaml

```diff
@@ -576,6 +576,22 @@ spec:
           - get
           - list
           - watch
+        - apiGroups:
+          - apps.openshift.io
+          resources:
+          - deploymentconfigs
```

Comment:
Do we want to add resources and discovery for deplyomentconfig? As of OpenShift Container Platform 4.14, DeploymentConfig objects are deprecated. 
Also if I understand it is only used to find the catalogs, right. In that case I think it would be better to use CatalogSource resource instead.

Reply:
@jakubschwan which version of OCP we are? If it's not too behind of 4.14, I'd rather keep it for now to support old instances.

Reply:
Clarified during a meeting - in shot, we are keeping this until we move to a version where it is removed.